.gitignore

```diff
@@ -0,0 +1,130 @@
+# Logs
```

Comment:
A great deal of these files don't exist so theres no point in them being in here, could easily be tidied up